full-stack-tests/core/src/backend/AzuriteTest.ts

```diff
@@ -0,0 +1,196 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { emptyDirSync, mkdirsSync } from "fs-extra";
+import { join } from "path";
+import * as azureBlob from "@azure/storage-blob";
+import { BlobContainer, CloudSqlite, IModelHost } from "@itwin/core-backend";
+import { AccessToken, Guid } from "@itwin/core-bentley";
+import { LocalDirName, LocalFileName } from "@itwin/core-common";
+
+// spell:ignore imodelid itwinid mkdirs devstoreaccount racwdl
+
+export namespace AzuriteTest {
+
+  export const storageType = "azure" as const;
+  export const httpAddr = "127.0.0.1:10000";
+  export const accountName = "devstoreaccount1";
+  export const baseUri = `http://${httpAddr}/${accountName}`;
+
+  export const getContainerUri = (id: string) => `${baseUri}/${id}`;
+  const pipeline = azureBlob.newPipeline(new azureBlob.StorageSharedKeyCredential(accountName, "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="));
```

Comment:
sorry, stupid q, but have to ask, this is not an actual valid/real account name & key right? i know this is testing against azurite, but having an actual account key looking string committed to src made me nervous